kernel/include/timeout_q.h

```diff
@@ -207,6 +207,17 @@ static inline void _add_timeout(struct k_thread *thread,
 	_dump_timeout(timeout, 0);
 	_dump_timeout_q();
 
+	/* If timer is submitted to expire ASAP with
+	 * timeout_in_ticks (duration) as zero value,
+	 * then handle timeout immedately without going
+	 * through timeout queue.
+	 */
+	if (!timeout_in_ticks) {
+		_handle_one_expired_timeout(timeout);
+		return;
+
```

Comment:
Minor thing: redundant empty line here

Reply:
Thanks @jhedberg / @carlescufi  for testing quickly :). appreciate your help.
Sure I will incorporate your review comment into next patch version.